app/models/post_image.rb

```diff
@@ -5,6 +5,12 @@ class PostImage < ActiveRecord::Base
 
   mount_uploader :image, PostImageUploader
 
+  def image_url
```

Comment:
Right, I agree with trying to not use try, but what about introducing a
null object then?
On Thu 15 Oct 2015 at 09:49 Miguel Palhas <notifications@github.com> wrote:

> In app/models/post_image.rb
> <https://github.com/subvisual/blog.subvisual.co/pull/108#discussion_r42097320>
> :
>
> > @@ -5,6 +5,12 @@ class PostImage < ActiveRecord::Base
> >
> >    mount_uploader :image, PostImageUploader
> >
> > +  def image_url
>
> yes, but I tend to try to avoid try whenever possible. And if you search
> for where I used this method, it helps clean up the mess that was hero &&
> hero.image && hero.image.url.present?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/subvisual/blog.subvisual.co/pull/108/files#r42097320>.
>